code/game/mecha/mecha_parts.dm

```diff
@@ -13,6 +13,9 @@
 	if(!user.transferItemToLoc(src, M))
 		to_chat(user, span_warning("\The [src] is stuck to your hand, you cannot put it in \the [M]!"))
 		return FALSE
+	if(occupant && istype(occupant.loc, /obj/item/mmi/posibrain)) // Cannot attach parts when someone is driving, unless it is a posi
```

Comment:
```suggestion
	if(istype(occupant?.loc, /obj/item/mmi/posibrain)) // Cannot attach parts when someone is driving, unless it is a posi
```

Reply:
This would not work, it needs to grab the occupant